dotCMS/src/integration-test/java/com/dotcms/util/IntegrationTestInitService.java

```diff
@@ -34,24 +35,28 @@ public static IntegrationTestInitService getInstance() {
     }
 
     public void init() throws Exception {
-        if (initCompleted.compareAndSet(false, true)) {
-            TestingJndiDatasource.init();
-            ConfigTestHelper._setupFakeTestingContext();
+        try {
+            if (initCompleted.compareAndSet(false, true)) {
+                TestingJndiDatasource.init();
+                ConfigTestHelper._setupFakeTestingContext();
 
-            CacheLocator.init();
-            FactoryLocator.init();
-            APILocator.init();
+                CacheLocator.init();
+                FactoryLocator.init();
+                APILocator.init();
 
-            //Running the always run startup tasks
-            StartupTasksUtil.getInstance().init();
+                //Running the always run startup tasks
+                StartupTasksUtil.getInstance().init();
 
-            //For these tests fire the reindex immediately
-            Config.setProperty("ASYNC_REINDEX_COMMIT_LISTENERS", false);
-            Config.setProperty("ASYNC_COMMIT_LISTENERS", false);
+                //For these tests fire the reindex immediately
+                Config.setProperty("ASYNC_REINDEX_COMMIT_LISTENERS", false);
+                Config.setProperty("ASYNC_COMMIT_LISTENERS", false);
 
-            Config.setProperty("NETWORK_CACHE_FLUSH_DELAY", (long) 0);
-            // Init other dotCMS services.
-            DotInitializationService.getInstance().initialize();
+                Config.setProperty("NETWORK_CACHE_FLUSH_DELAY", (long) 0);
+                // Init other dotCMS services.
+                DotInitializationService.getInstance().initialize();
+            }
+        } catch(Exception e) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid catching generic exceptions such as NullPointerException, RuntimeException, Exception in try-catch block](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=3712629)